src/pe_reports/stakeholder/views.py

```diff
@@ -0,0 +1,595 @@
+            verifiedIP.append(the_ip)
+    return verifiedIP
+
+
+def getOrganizations():
+    """Get all orgaization details from Cybersix Gill via API."""
+    url = "https://api.cybersixgill.com/multi-tenant/organization"
+
+    headers = {
+        "Content-Type": "application/json",
+        "Cache-Control": "no-cache",
+        "Authorization": f"Bearer {getToken()}",
+    }
+
+    response = requests.get(url, headers=headers).json()
+    return response
+
+
+def setNewCSGOrg(newOrgName, orgAliases, orgdomainNames, orgIP, orgExecs):
+    """Set a new stakeholder name at CSG."""
+    newOrganization = json.dumps(
+        {
+            "name": f"{newOrgName}",
+            "organization_commercial_category": "customer",
+            "countries": ["worldwide"],
+            "industries": ["Government"],
+        }
+    )
+    url = "https://api.cybersixgill.com/multi-tenant/organization"
+
+    headers = {
+        "Content-Type": "application/json",
+        "Cache-Control": "no-cache",
+        "Authorization": f"Bearer {getToken()}",
+    }
+
+    response = requests.post(url, headers=headers, data=newOrganization).json()
+
+    newOrgID = response["id"]
+
+    if newOrgID:
+        logging.info("Got here there is a new new org %s", newOrgID)
+
+        setOrganizationUsers(newOrgID)
+        setOrganizationDetails(newOrgID, orgAliases, orgdomainNames, orgIP, orgExecs)
+
+    return response
+
+
+def setOrganizationUsers(org_id):
+    """Set CSG user permissions at new stakeholder."""
+    role1 = os.getenv("USERROLE1")
+    role2 = os.getenv("USERROLE2")
+    id_role1 = os.getenv("USERID")
+    csg_role_id = os.getenv("CSGUSERROLE")
+    csg_user_id = os.getenv("CSGUSERID")
+    for user in getalluserinfo():
+        userrole = user[csg_role_id]
+        user_id = user[csg_user_id]
+
+        if (
+            (userrole == role1)
+            and (user_id != id_role1)
+            or userrole == role2
+            and user_id != id_role1
+        ):
+
+            url = (
+                f"https://api.cybersixgill.com/multi-tenant/organization/"
+                f"{org_id}/user/{user_id}?role_id={userrole}"
+            )
+
+            headers = {
+                "Content-Type": "application/json",
+                "Cache-Control": "no-cache",
+                "Authorization": f"Bearer {getToken()}",
+            }
+
+            response = requests.post(url, headers=headers).json()
+            logging.info(response)
+
+
+def setOrganizationDetails(org_id, orgAliases, orgDomain, orgIP, orgExecs):
+    """Set stakeholder details at newly created.
+
+    stakeholder at CSG portal via API.
+    """
+    logging.info("The following is from setting details")
+    logging.info("The org_id id %s", org_id)
+    logging.info("The org_id id %s", orgAliases)
+    logging.info("The org_id id %s", orgDomain)
+    logging.info("The org_id id %s", orgIP)
+    logging.info("The org_id id %s", orgExecs)
```

Comment:
I think this is what you were going for?
```suggestion
    logging.info("The org_id is %s", org_id)
    logging.info("The orgAliases is %s", orgAliases)
    logging.info("The orgDomain is %s", orgDomain)
    logging.info("The orgIP is %s", orgIP)
    logging.info("The orgExecs is %s", orgExecs)
```